hydra-node/hydra-node.cabal

```diff
@@ -129,6 +130,7 @@ library
     , contra-tracer
     , data-default
     , filepath
+    , formatting
```

Comment:
Can you give a quick explanation / intro about this package? Never used it and also would be interested what for we use it now.

Reply:
Used to better format CBOR deserialisation error. The cardano-base package provides `Buildable` instance for the `DecoderError` which are nicer than the default `Show`... ideally, I wish `cardano-base` exposed a `prettyDecoderError` function so we wouldn't have to import this dependency.

This is debatable, we can always fallback to `show`.